lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/TreeDataCsvTransformer.scala

```diff
@@ -333,12 +356,24 @@ case class TreeDataCsvTransformer() {
           } else {
             cols = line.split(",", -1).map(_.trim)
           }
-          val Tree = lineProcessor.process(line)
-          treeList += Tree
-          treeMap += (Tree.id -> Tree)
+          val tree = lineProcessor.process(line)
+          sink.accept(tree)
+          treeList += tree
```

Comment:
I don't think you really need treeList or treeMap, do you? They're just for testing. If you can keep a count that's good enough, do that.